src/gt4py/next/ffront/decorator.py

```diff
@@ -114,12 +145,9 @@ def past_stage(self):
     def __post_init__(self):
         if self.backend is not None and self.backend.transforms_prog is not None:
             self.backend.transforms_prog.past_lint(self.past_stage)
+        _frozen_update_wrapper(self, self.definition)
         return next_backend.DEFAULT_PROG_TRANSFORMS.past_lint(self.past_stage)
 
-    @property
-    def __name__(self) -> str:
-        return self.definition_stage.definition.__name__
-
```

Comment:
This works because `__name__` is in the default `functools.WRAPPER_ASSIGNMENTS` value for the `assigned` arg of `_frozen_update_wrapper()`, right?